waypoint_generator/CMakeLists.txt

```diff
@@ -0,0 +1,211 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(waypoint_generator)
+
+## Compile as C++14 as required by lanelet
+add_compile_options(-std=c++14)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  autoware_msgs
+  cav_msgs
+  cav_srvs
+  carma_utils
+  carma_wm
+  roscpp
+  std_msgs
+)
+
+## System dependencies are found with CMake's conventions
```

Comment:
Feel free to remove those commented lines just for better reading experience.

Reply:
Will do